seed/migrations/0125_dq_refactor.py

```diff
@@ -0,0 +1,98 @@
+# Generated by Django 2.2.10 on 2020-04-13 23:03
+
+from django.db import migrations, models
+
+
+def forwards(apps, schema_editor):
+    Rule = apps.get_model('seed', 'Rule')
+    # data_type not null:
+    #     data_type is 1:
+    #         text_match null: not_null -> not_null;
+    #                         !not_null -> required: required;
+    #                                   -> !required -> '';
+    #         text_match not null: include -> not_null: (+)not_null;
+    #                                      -> !not_null -> required: (+)required;
+    #     data_type is not 1 (0,2,3,4,5):
+    #         min and max null: not_null -> not_null;
+    #                         : !not_null -> required: required;
+    #                                     -> !required: '';
+    #         min or max not null: range -> not_null: (+)not_null;
+    #                                    -> !not_null -> (+)required: required;
+    # data_type null:
+    #     min and max null -> text_match null: not_null -> not_null;
+    #                                         !not_null -> required: required;
+    #                                                   -> !required: '';
+    #                      -> text_match not null: include -> not_null: (+)not_null;
+    #                                                      -> !not_null -> required: (+)required;
+    #     min or max not null: range -> not_null: (+)not_null;
+    #                                -> !not_null -> required: (+)required;
+
+    def create_condition(rule, condition):
+        Rule.objects.create(name=rule['name'], description=rule['description'], table_name=rule['table_name'], field=rule['field'],
+                            enabled=rule['enabled'], data_type=rule['data_type'], rule_type=rule['rule_type'], min=rule['min'],
+                            max=rule['max'], severity=rule['severity'], units=rule['units'],
+                            data_quality_check_id=rule['data_quality_check_id'], status_label_id=rule['status_label_id'],
+                            text_match=rule['text_match'], condition=condition)
+
+    Rule.objects.filter(data_type=1, text_match=None, not_null=True).update(condition='not_null')
+    Rule.objects.filter(data_type=1, text_match=None, required=True, not_null=False).update(condition='required')
+    Rule.objects.filter(data_type=1, text_match=None, required=False, not_null=False).update(condition='')
```

Comment:
For cases like this where we're setting `condition=''`, would it make sense to go disable these for users to further make the point that these aren't doing anything and requires action?